src/Layout/Footer.tsx

```diff
@@ -193,8 +195,33 @@ function LanguageModal({
 }: ModalProps): ReactElement {
   const { theme } = React.useContext(ThemeContext);
 
-  const { language, setlanguage } = React.useContext(QueryParamContext);
+  const {
+    language,
+    setlanguage,
+    projectDetails,
+    setshowErrorCard,
+    tenant,
+    country,
+    setProjectName,
+    setProjectDescription,
+  } = React.useContext(QueryParamContext);
   const { t, ready } = useTranslation(["common"]);
+  async function loadPaymentSetup() {
```

Comment:
We have now three times a function named `loadPaymentSetup`. Also at https://github.com/Plant-for-the-Planet-org/planet-donations/blob/a0135c13e63b114159dd4f1e7e30206a3f29cc0b/src/Layout/QueryParamContext.tsx#L255 and at https://github.com/Plant-for-the-Planet-org/planet-donations/blob/a0135c13e63b114159dd4f1e7e30206a3f29cc0b/src/Donations/Micros/PaymentStatus/FailedDonation.tsx#L44 - all differ from each other, but generally doing the same API call. That's bad style.